### PR TITLE
feat: verify redesign, and misc refactorings

### PR DESCRIPTION
**Pull request by cavacado**
looking for comments

this pr is huge,

it does:
1) moves the verify component into its own route
2) changes the design for the current verify page
3) refactors some ways we are using redux (move towards hooks method)
4) refactors some js files into ts to leverage on static typing

TODO:
- implement the try another way for mobile verify

### CODE REVIEW

File: src/test/demo-sample.spec.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
   await validateIframeTexts(["Name & Address of Shipping Agent/Freight Forwarder"]);
 });
 
-test("demo sample document is rendered correctly when clicked", async (t) => {
+// TODO, implement
+// eslint-disable-next-line jest/no-disabled-tests
+test.skip("demo sample document is rendered correctly when clicked", async (t) => {
+  await navigateToVerify();
   await t.resizeWindow(800, 600);
   await t.click(Selector("#demoClick"));
```
Comment (simboonlong) [inline src/test/demo-sample.spec.js:18]: redesign no longer has this mobile button yeah? can remove this test i guess?

File: src/components/VerifyPageContent/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const roundInstructionsText = "Drag the document over to see a demo of TradeTrust features";
```
Comment (simboonlong) [inline src/components/VerifyPageContent/index.ts:1]: same, to enum easier to add more messages later also

File: src/components/CertificateDropZone/Views/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const unverifiedMessage = "What Should I Do";
+export const docNotValidMessage = "This document is not valid";
+export const tryAnotherMessage = "Try another document";
```
Comment (simboonlong) [inline src/components/CertificateDropZone/Views/index.ts:3]:
```suggestion
export enum VerifyMessage {
  UNVERIFIED = "What Should I Do",
  INVALID = "This document is not valid",
  RETRY = "Try another document"
}
```

this way could make use of the autocomplete to "see" all states

File: src/test/demo-sample.spec.js (hunk 1 above)
Comment (cavacado) [reply src/test/demo-sample.spec.js:18]: acutally it does, i just havent added it in yet
Screenshot 2021-06-23 at 9 37 26 AM

File: src/components/CertificateDropZone/Views/index.ts (hunk 3 above)
Comment (cavacado) [reply src/components/CertificateDropZone/Views/index.ts:3]: hmm, i could do that, but this is mainly to prep for future use case of moving towards localisation.

actually would it be smart to just have a file with all our texts in them and organised properly?

something like `messages/en.ts`

```ts
export default text = {
  [<component>-<message-description>]: "message text",
  "verify-unverify": "What Should I Do",
  "verify-invalid": "This document is not valid",
}
....
```

**Review by simboonlong — APPROVED** [review]

**Pull request merged**